### DISCUSSION TITLE
Welcome to install-jdk Discussions!

### DISCUSSION THREAD

**Discussion by jyksnw** [jyksnw/install-jdk#46] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other community members. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To start, comment below with an introduction and tell us about what you do with this community.

**Comment by jcdevil** [comment]
Hello ! Is this project still alive ? Is there a chance that pending PR will be merged in the coming weeks ?

**Reply by jyksnw** [reply]
Hi @jcdevil thanks for inquiring. This project is still alive and actively used by many. New PRs are welcome against open issues and feature requests. Please feel free to create your request following the contributing guidelines found at https://github.com/jyksnw/install-jdk/blob/master/CONTRIBUTING.MD

I look forward to collaborating with you on any improvements you may have.
